From: zhangweibing <zhangweibing@unitedstack.com>
Date: Fri, 26 Aug 2016 10:49:48 +0800
Subject: rgw: remove duplicated calls to getattr

Signed-off-by: Weibing Zhang <zhangweibing@unitedstack.com>
(cherry picked from commit 87b550ed2fba472d4ef2e1f771bb5f9fd248804f)

Fixes: http://tracker.ceph.com/issues/17327
Resolves: rhbz#1381684
---
 src/rgw/rgw_file.cc | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/rgw/rgw_file.cc b/src/rgw/rgw_file.cc
index 918b942..b24060b 100644
--- a/src/rgw/rgw_file.cc
+++ b/src/rgw/rgw_file.cc
@@ -1329,12 +1329,6 @@ int rgw_getattr(struct rgw_fs *rgw_fs,
   RGWLibFS *fs = static_cast<RGWLibFS*>(rgw_fs->fs_private);
   RGWFileHandle* rgw_fh = get_rgwfh(fh);
 
-  int rc = -(fs->getattr(rgw_fh, st));
-  if (rc != 0) {
-    // do it again
-    rc = -(fs->getattr(rgw_fh, st));
-  }
-
   return -(fs->getattr(rgw_fh, st));
 }
 
